src/tpm2-crypto.c

```diff
@@ -529,13 +529,26 @@ static UsefulBuf concat_data_sign(UsefulBuf data, UsefulBuf sign, UsefulBuf buf)
     }
 }
 
-UsefulBuf sign_with_aik(UsefulBuf data)
+UsefulBuf sign_with_aik(UsefulBuf data, UsefulBuf nonce)
```

Comment:
```suggestion
UsefulBuf sign_with_aik(UsefulBuf data, UsefulBufC nonce)
```
Plus drop `UsefulBuf_Unconst()` from calls to this function.

Reply:
Fixed